README.md

```diff
@@ -592,6 +592,23 @@ Example payload:
 }
 ```
 
+#### Authorization revoked
+
+User can revoke an application access at any given time without prior notice trough Fractal ID dashboard. Upon getting `authorization_revoked` notification, partner can be notified that the user has decided not to share his information from now on. It means that all access grants and access tokens are being revoked and partner won't be able to query this user anymore.
```

Comment:
Few changes:
- `trough -> through`
- `share his` -> `share their`
- "Upon getting `authorization_revoked` notification, partner can be notified that the user has decided not to share their information from now on" -> "Fractal ID will emit a `authorization_revoked` notification when this happens."
- `are being` -> `were`? By the time they get the notification, this should all have happened, right?

Reply:
Updated, thanks for the input 💯